ui/src/ducks/app/nodes.js

```diff
@@ -484,19 +484,27 @@ export function* stopRefreshNodes() {
 
 export function* fetchNodesIPsInterface() {
   let result;
+
+  // Call K8s API to get the existing node to fetch grains
+  yield call(fetchNodes);
```

Comment:
I think fetchNodesIPsInterface should. be called when fetching nodes is finished instead of re-triggering an API call to kubernetes API